cmd/analyzer/main.go

```diff
@@ -56,14 +58,14 @@ func _main(cmdlineArgs []string) error {
 		return fmt.Errorf("error generating cloud config from input vpc resources file: %w", err)
 	}
 
-	og, err := vpcmodel.NewOutputGenerator(cloudConfig, *inArgs.Grouping, analysisTypeToUseCase(inArgs))
-	if err != nil {
-		return err
-	}
 	outFile := ""
 	if inArgs.OutputFile != nil {
 		outFile = *inArgs.OutputFile
 	}
+	og, err := vpcmodel.NewOutputGenerator(cloudConfig, *inArgs.Grouping, analysisTypeToUseCase(inArgs), outFile != ARCHDRAWIOFormat)
```

Comment:
why comparing outFile and not the outputFormat to `ARCHDRAWIOFormat` ?

Reply:
fixed